pkg/project/types/types.go

```diff
@@ -1,9 +1,9 @@
 package types
 
 type Script struct {
-	API    string `toml:"api"`
-	Inline string `toml:"inline"`
-	Shell  string `toml:"shell"`
+	API    string `toml:"api,omitempty"`
+	Inline string `toml:"inline,omitempty"`
+	Shell  string `toml:"shell,omitempty"`
```

Comment:
I can also create a different type in the order.go file instead with the omitempty options if here it's not the place.